databricks_cli/cli.py

```diff
@@ -67,6 +68,7 @@ def cli():
 cli.add_command(instance_pools_group, name="instance-pools")
 cli.add_command(pipelines_group, name='pipelines')
 cli.add_command(repos_group, name='repos')
+cli.add_command(unity_catalog_group, name='unity-catalog')
```

Comment:
for consistent CLI UX, we should rather add command groups of catalogs, creds, sharing, external locations, metastore, permissions, schemas, and tables directly here. Plus other consistency comments below.

Reply:
Addressed in #471.